### DISCUSSION TITLE
Is there any way to check fares when there is a companion or minor booked on the reservation?

### DISCUSSION THREAD

**Discussion by emcrouth** [jdholtz/auto-southwest-check-in#195] [General]
I still check my fares when there's a companion or minor linked..  takes a little work to rebook (as you need to cancel the companion or minor then rebook after adjustment) but functionality would be super helpful.  Thanks!

**Comment by jdholtz** [comment]
Hello. This question has already been answered [here](https://github.com/jdholtz/auto-southwest-check-in/discussions/87#discussioncomment-5835874). AFAIK, it is not possible to check fares with a companion without either manually entering the price you paid (there is no automatic way to get this from the Southwest website) and comparing it to the flight price now or cancelling the reservation, checking the fare, and rebooking the reservation (which I have no intention of doing in this script).

**Reply by emcrouth** [reply]
Thanks - I was hoping there was a way to use the fare checker and just manually enter the date, flight, and price paid, so it could check vs. trying to pull that info direct from the change form.  I don't believe that functionality exists in the current version.

I always update my flights manually anyway if there's a price change...  and FYI - I've found that if you have duplicate flights book (so if I book an extra flight just for price checking to cancel in the future, Southwest's system now cancels your duplicate flight).

**Reply by jdholtz** [reply]
> I don't believe that functionality exists in the current version.

You are correct. I also don’t want to support manual intervention in the script as it is intended to be run fully automatically. More discussion on that point can be found [here](https://github.com/jdholtz/auto-southwest-check-in/pull/78#issuecomment-1518354334).
